templates/ai/index.html

```diff
@@ -0,0 +1,533 @@
+                    alt="",
+                    width="600",
+                    height="400",
+                    hi_def=True,
+                    attrs={"class": "p-image-container__image"},
+                    loading="lazy") | safe
+          }}
+        </div>
+        <p>
+          Ubuntu 是数据科学家的首选操作系统。在 Ubuntu 工作站上开发机器学习模型，并部署到 Ubuntu 服务器。无论是在云端还是边缘（edge），整个 AI 生命周期都能享受一致且熟悉的 Ubuntu 体验。
+        </p>
+        <p>
+          而生产级项目更需要一个能够实现可扩展、可移植和可重制的解决方案。Canonical MLOps 加速 AI 项目的开发进程，为您提供：
+        </p>
+        <ul class="p-list--divided">
+          <li class="p-list__item has-bullet">在任何云环境中，无论是私有云还是公共云，都能获得相同的体验</li>
+          <li class="p-list__item has-bullet">低运维、简化的生命周期管理</li>
+          <li class="p-list__item has-bullet">一套模块化的开源工具，支持可重用的部署</li>
+        </ul>
+        <div class="p-cta-block">
+          <a href="/blog/edge-ai_cn" class="p-button--positive">边缘 AI 用例与实践</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="p-rule is-fixed-width" />
+
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <h2>
+        使用开源技术在边缘部署
+        <br />
+        <a href="https://ubuntu.com/engage/guide-to-deploying-ai-at-the-edge">深入了解 Edge AI&nbsp;&rsaquo;</a>
+      </h2>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row">
+      <hr class="p-rule--highlight" />
+      <div class="p-section--shallow">
+        <h2>企业级开源 AI 服务及支持</h2>
+      </div>
+      <hr class="p-rule" />
+      <div class="row--50-50">
+        <div class="col">
+          <h3 class="p-heading--4">MLOps 托管服务</h3>
+          <p>
+            基础架构交由 Canonical 专家管理，您可放心专注于构建生产级模型。
+          </p>
+          <ul class="p-list--divided">
+            <li class="p-list__item is-ticked">99.9% uptime</li>
+            <li class="p-list__item is-ticked">24/7 全天监控</li>
+            <li class="p-list__item is-ticked">高可用性</li>
+          </ul>
+          <div class="col p-section--shallow u-hide--medium u-hide--large">
+          <p>
+            <a href="https://pages.ubuntu.com/rs/066-EOV-335/images/Charmed%20Kubeflow%20-%20Digital.pdf">MLOps 托管手册&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        </div>
+        <div class="col">
+          <h3 class="p-heading--4">AI 咨询服务</h3>
+          <p>
+            由 Canonical 專家帶領更深入地理解您的数据，成功实现您的用例。
+          </p>
+          <ul class="p-list--divided">
+            <li class="p-list__item is-ticked">数据探索 workshop</li>
+            <li class="p-list__item is-ticked">Canonical MLOps 部署</li>
+            <li class="p-list__item is-ticked">MLOps 实作 workshop</li>
+            <li class="p-list__item is-ticked">基于 PoC 实施</li>
+          </ul>
+        </div>
+      </div>
+      <div class="row--50-50">
+        <div class="col u-hide--small">
+          <p>
+            <a href="https://pages.ubuntu.com/rs/066-EOV-335/images/Charmed%20Kubeflow%20-%20Digital.pdf">MLOps 托管手册&nbsp;&rsaquo;</a>
+          </p>
```

Comment:
For this and the one below please
```suggestion
          <div class="p-cta-block is-borderless">
            <a href="https://pages.ubuntu.com/rs/066-EOV-335/images/Charmed%20Kubeflow%20-%20Digital.pdf">MLOps 托管手册&nbsp;&rsaquo;</a>
          </div>
```